source/_docs/ecosystem/nginx.markdown

```diff
@@ -111,6 +111,7 @@ http {
 
         # These shouldn't need to be changed
         listen 443 default_server;
+        listen [::]:443 default_server ipv6only=on;
```

Comment:
As per https://trac.nginx.org/nginx/ticket/345 we might need to set `ipv6only=off` then.

When running Jessie the necessary nginx version is only available in the backports I'm afraid.